src/applications/coronavirus-screener/components/FormQuestion.jsx

```diff
@@ -7,6 +9,12 @@ export default function FormQuestion({
   scrollNext,
 }) {
   function handleChange(event) {
+    if (_.isEmpty(formState)) {
```

Comment:
alternatively checking `formState == {}` would probably do the same thing?

Reply:
Tested this and does not seem to work.  isEmpty seems like the reliable way to do this check. LMK if I am missing something on this.

Reply:
hmm interesting! i'm not against `isEmpty` but im curious why it wouldn't work the other way

Reply:
ah i see there are some various ways to do this in vanilla JS, agree though this is one of those cases where it's best to use a util lib.